modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDynamicParameterTest.java

```diff
@@ -82,6 +82,9 @@ public void testDynamicParameters() {
         assertQuery("SELECT ? % ?").withParams(11, 10).returns(1).check();
         assertQuery("SELECT ? + ?, LOWER(?) ").withParams(2, 2, "TeSt").returns(4, "test").check();
         assertQuery("SELECT LOWER(?), ? + ? ").withParams("TeSt", 2, 2).returns("test", 4).check();
+        // TODO: returns incorrect result, need to be fixed in scope of https://issues.apache.org/jira/browse/IGNITE-20784
+        assertQuery("SELECT (2147483647 + ?)::INTEGER").withParams(1).returns(-2147483648).check();;
+        assertQuery("SELECT (? + 2147483647)::INTEGER").withParams(1).returns(-2147483648).check();;
```

Comment:
What is the purpose of writing invalid test?
It would be better writing a commented code with expected result.